en/docs/reference/mediators/data-mapper-mediator.md

```diff
@@ -174,6 +174,17 @@ The operations that the Data Mapper supports as shown below.
 
 - **match** – check whether the input match with a (JS) Regular Expression
 
+## AI Data Mapper
+The MI Copilot AI Assistant will allow you to seamlessly generate the input-output mapping. Simply load the input and output schema (of any type) to the relevant sections as shown below and click **Map**. A **Clear** button is also available for convenience. 
```

Comment:
Instead of `also available for convenience`, shall we mention what the clear btn does?